elide-core/src/main/java/com/yahoo/elide/core/filter/expression/InMemoryFilterVisitor.java

```diff
@@ -5,23 +5,23 @@
  */
 package com.yahoo.elide.core.filter.expression;
 
-import com.yahoo.elide.core.EntityDictionary;
+import com.yahoo.elide.core.RequestScope;
 
 import java.util.function.Predicate;
 
 /**
  * Visitor for in memory filterExpressions
  */
 public class InMemoryFilterVisitor implements Visitor<Predicate> {
-    private final EntityDictionary dictionary;
+    private final RequestScope requestScope;
 
-    public InMemoryFilterVisitor(EntityDictionary dictionary) {
-        this.dictionary = dictionary;
+    public InMemoryFilterVisitor(RequestScope requestScope) {
```

Comment:
I don't think so. This is already located in `elide-core`, so I see little reason to take the more generic interface. That interface seems to be more limiting and everywhere we use it, it gets casted to `elide-core`'s version anyway.

Generally speaking, if someone was to overload RequestScope, they would really have to extend `elide-core`'s `RequestScope`. If they didn't we would likely get cast exceptions in many places.